bundle/edu.gemini.util.osgi/build.sbt

```diff
@@ -24,8 +27,7 @@ OsgiKeys.bundleActivator := None
 
 OsgiKeys.bundleSymbolicName := name.value
 
-OsgiKeys.dynamicImportPackage := Seq("")
+OsgiKeys.dynamicImportPackage := Seq("*")
```

Comment:
We need this because service lookup is now done in a static method on `SecureServiceFactory`.